### ISSUE TITLE
Make PDF output accessible

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#5558]
Following #1498/#5342, we should be able to produce output that is accessible. Right now, this is still not the case:

- MacOS Preview doesn't read any content
- Adobe Acrobat only reads links inside the document

However, we now have working bookmarks/TOC (#5291).

Next step: check PDF renderer APIs to see what we need to do to make the content accessible.
